### PR TITLE
Starostin-2017-401

### PR DESCRIPTION
**Pull request by CarolinHu**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

The list from issue #378.
@Kristina-Pianykh, I think a speaker of Russian would be good here, would you mind being the moderator?

### CODE REVIEW

File: concepticondata/references/references.bib [hunk 1]
```diff
@@ -3604,3 +3604,10 @@ @Article{List2020
   Date                                = {2020-10-19},
   Url                                 = {https://calc.hypotheses.org/2545}
 }
+
+@Misc{Starostin2017,
```
[the remark this answers is not in the document]
Comment (CarolinHu) [reply concepticondata/references/references.bib:3608]: And it's right at the top as well, no idea how I could overlook it. Will do!

File: concepticondata/conceptlists/Starostin-2017-401.tsv [hunk 2]
```diff
@@ -0,0 +1,402 @@
+ID	NUMBER	ENGLISH	RUSSIAN	SEMANTIC_LINKS	CONCEPTICON_ID	CONCEPTICON_GLOSS
```
[the remark this answers is not in the document]
Comment (CarolinHu) [reply concepticondata/conceptlists/Starostin-2017-401.tsv:1]: No it's fine, I can add that column as well.

**Comment by LinguList** [comment]
nice we do the review than after this has been added. Thanks!

**Review by LinguList — APPROVED** [review]
For me, this is very nice in this form, so I consider it good to go. I do not really think that it is needed to have somebody else look at this, as long as the basic tests have passed.

**Comment by LinguList** [comment]
So feel free to merge, if you think it is okay, @CarolinHu

**Comment by CarolinHu** [comment]
Okay, thank you for reviewing!

**Pull request merged**